instrumentation/mysql/src/main/java/brave/mysql/aws/XRayTracingStatementInterceptor.java

```diff
@@ -0,0 +1,42 @@
+package brave.mysql.aws;
+
+import brave.Span;
+import brave.mysql.TracingStatementInterceptor;
+import com.mysql.jdbc.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class XRayTracingStatementInterceptor extends TracingStatementInterceptor {
+
+  private final static Logger LOGGER = Logger.getLogger(XRayTracingStatementInterceptor.class.getName());
+
+  private final boolean useQueryForUrl;
+
+  public XRayTracingStatementInterceptor() {
+    this(false);
+  }
+
+  public XRayTracingStatementInterceptor(boolean useQueryForUrl) {
+    this.useQueryForUrl = useQueryForUrl;
+  }
+
+  @Override
+  protected void doPreProcessTag(Connection connection, String sql, Span span) {
```

Comment:
you've probably seen the http client parser stuff in brave-instrumentation-http. This hints at a similar one brave-instrumentation-sql. If you are up to it, probably a good thing to make a module as most of these properties will be re-used across p6spy and the two mysql variants. This will reduce the amount of copy/pasting and also decouple dependencies when you go to make brave-parser-xray